python/interpret_community/mimic/model_distill.py

```diff
@@ -5,6 +5,7 @@
 """Utilities to train a surrogate model from teacher."""
```

Comment:
thinking about this more, we shouldn't handle this in model_distill function but in the model wrapper.  We can have a wrapper for models that output pandas dataframes for predict/predict_proba.

Reply:
Thanks for the suggestion. I am wrapping the models giving dataframe as outputs for predict/predict_proba and converting the outputs into numpy arrays

Reply:
Changed to handle the inside the wrapped models for classification and regression.